go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/containerd/console v1.0.3
 	github.com/containerd/containerd v1.7.0-beta.3
 	github.com/containerd/continuity v0.3.0
-	github.com/containerd/go-cni v1.1.7
+	github.com/containerd/go-cni v1.1.8-0.20221026181309-f108694a5873
```

Comment:
Planning to release v1.1.8 soon
- https://github.com/containerd/go-cni/pull/109

Reply:
Roger, this code only depends on this PR: https://github.com/containerd/go-cni/pull/103

I will rebase when the deps get updated.

Reply:
Released v1.1.8